docs/sql-guide/functions/function-parsetimestamp.md

```diff
@@ -0,0 +1,138 @@
+---
+title: PARSETIMESTAMP()
+layout: default
+parent: SQL functions
+grand_parent: SQL guide
+---
+## Before you begin
+* [Learn about unix epoch/unix time](https://en.wikipedia.org/wiki/Unix_time){:target="_blank"}
+
+# PARSETIMESTAMP() function
+
+The `PARSETIMESTAMP()` function converts the input time value in string format to timestamp based on the format provided.
+
+## Syntax
+
+```
+PARSETIMESTAMP(expr, format)
+```
+
+## Arguments
+
+| Argument | Data type | Description | Required | Further information |
+|---|---|---|---|---|
+| expr   | string | Input string values that needs to be converted to a timestamp | Yes |
+| format | string | String value that specifies the format used to parse the time value. | Yes | [Format Codes](#format-codes) |
+
+
+## Format Codes
```

Comment:
Are these format codes explained outside (e.g., in the Unix timestamp page linked under the before you begin heading)?

If so it's ok to just refer the user out to them. In general, it's best not to duplicate content already documented, especially if there's an existing page :)

Reply:
This is the subset we support, so I believe it's valid to put them here